packages/schematics/angular/migrations/update-8/drop-es6-polyfills.ts

```diff
@@ -45,12 +56,20 @@ const header = `/**
 /***************************************************************************************************
 * BROWSER POLYFILLS
 */
-
-/** IE9, IE10 and IE11 requires all of the following polyfills. **/
-// import 'core-js/es6/weak-map';
 `;
 
 const applicationPolyfillsHeader = 'APPLICATION IMPORTS';
+const browserPolyfillsHeader = 'BROWSER POLYFILLS';
+
+const knownPolyfillHashes = [
+  '3dba718d7afe009e112e10d69073d2a2', // 6.0 - unmodified
+  'fccdb76b06ea636933f8b99b1c8d9725', // 6.0 - all core-js uncommented
+  '97e16639be1de06153695f5fefde745d', // 7.0 - unmodified
+  'd6c13d6dcf94ff3749283f33dd0d4864', // 7.0 - all core-js uncommented
+  '79bf0fd46c215e5f4145e15641c325f3', // 7.2 - unmodified
+  '6fe8080c7e38ee0ce677fdbc3884377a', // 7.2 - all core-js uncommented
+  '8e7f6abb3d2dca03b4dbb300e400a880', // 7.3 - unmodified
```

Comment:
will we have to maintain this in the future? please add something to the right doc so we remember to keep it updated